extensions/amp-ad-exit/amp-ad-exit.md

```diff
@@ -273,8 +276,7 @@ Example:
         },
         "_3pAnalytics": {
           "defaultValue": "no_response",
-          "vendorAnalyticsSource": "VendorXYZ",
-          "vendorAnalyticsResponseKey": "findings"
+          "iframeTransportSignal": "IFRAME_TRANSPORT_SIGNAL(example-3p-vendor,collected-data)"
```

Comment:
LGTM. You might add a note that (at least for now) adding a space after this comma is not allowed. (See https://github.com/ampproject/amphtml/pull/11461#discussion_r142504598 )

Reply:
Thanks, done

Reply:
(Note that the comma thing is no longer true...)

Reply:
@calebcordry is working on nested function calls in URL replacement. we might again restrict the space use here.

Reply:
I am not 100% yet. There might be a way to preserve spacing after comma only. I am still thinking it through.